components/brave_wallet/browser/solana_provider_impl.h

```diff
@@ -83,8 +93,10 @@ class SolanaProviderImpl final : public mojom::SolanaProvider,
   // will be saved and future connect from the same site will not ask user for
   // permission again until the permission is removed.
   base::flat_set<std::string> connected_set_;
+  int sign_message_id_ = 0;
```

Comment:
As mentioned in DM, I think we should probably fill  the request ID in `BraveWalletService` instead for both ethereum and solana dapp requests when they're added via `AddSignMessageRequest`.

Reply:
addressed in https://github.com/brave/brave-core/pull/13419/commits/7cbe350c7523c8acfdade7740ca02421c5fe6f34